test/gas.js

```diff
@@ -272,5 +272,92 @@ describe("Gas", function() {
       let gasPrice = '0x2000'
       return testGasExpenseIsCorrect(gasPrice, false, new Web3(Ganache.provider({ mnemonic, gasPrice })))
     })
-  })
+    it('should calculate cumalativeGas and gasUsed correctly when multiple transactions are in a block', function (done){
+      let tempWeb3 = new Web3(Ganache.provider({
+        blockTime: .5, // seconds
+        mnemonic: "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
+      }));
+
+      const accounts = [ // configured with mnemonic
+        '0x627306090abab3a6e1400e9345bc60c78a8bef57',
+        '0xf17f52151ebef6c7334fad080c5704d77216b732',
+      ];
+      const secretKeys = [ // configured with mnemonic
```

Comment:
I don't see `secretKeys` used anywhere. Perhaps this was just left over from debugging?